docs/book/save-handler.md

```diff
@@ -72,32 +72,6 @@ $manager      = new SessionManager();
 $manager->setSaveHandler($saveHandler);
 ```
 
-## MongoDB
```

Comment:
The removal must be covered in a migration guide, under the headline "Removed Features".
A good example can be found in the documentation of laminas-validator: ["Migration from Version 2 to 3"](https://docs.laminas.dev/laminas-validator/v3/migration/v2-to-v3/)